parlai/crowdsourcing/tasks/turn_annotations_static/webapp/README.md

```diff
@@ -0,0 +1,16 @@
+## Description
```

Comment:
@JackUrb Do you know if this README is up-to-date? I had a similar one that you said was obsolete

Reply:
Yeah unfortunately this is out-of-date. It's referencing some early ParlAI-MTurk code, and I need to remove it from the example in Mephisto. (or update it, rather)

Reply:
Removing as per conversation with @EricMichaelSmith that @JackUrb said to do that on another task.